### DISCUSSION TITLE
Whether we can add a third sidebar for VSCode?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#368] [Extension Development QnA] [answered]
We know VSCode's UI is like below,
![image](https://user-images.githubusercontent.com/12727582/210679640-ac0f44a7-d61f-4d32-a304-97e57ba7031b.png)

 I want to know whether we can add a third Sidebar for it.

**Comment by alefragnani** [comment, marked as answer]
Hi @liaodalin19903 ,

No, you can't. There are only two Sidebars available.

Hope this helps

**Comment by user1007017** [comment]
why only two sidebars, if on a extra wide screen easily 4 sidebars could fit?

**Reply by user1007017** [reply]
ok, CTRL+P "Move Terminal into New Window" did what I wanted
